compute/admin_guide/install/system_requirements.adoc

```diff
@@ -434,14 +459,62 @@ ECS agent 1.61.0
 
 Docker 20.10.7
 
-|Kubernetes
+|Kubernetes with Docker
 |Docker Engine version: 20.10.14
 
 API version:1.41
 
 Go Version: go1.16.15
 
-Native Kubernetes 1.23.5 (containerd 1.5.11)
+|Kubernetes with containerd
+|Kubernetes 1.23.5 (containerd 1.5.11)
+
+|Lightweight Kubernetes (k3s)
+|k3s version: 1.23.8+k3s2 (containerd 1.5.13-k3s1)
+
+k3s version: v1.23.6+k3s1 (containerd 1.5.11-k3s2)
+
+k3s version: v1.23.8+k3s1 (containerd 1.5.13-k3s1)
+
+|GKE on ARM
+|GKE version: v1.24.1-gke.1400
+
+containerd version: 1.6.2
+
+Kernel version: 5.10.107+
+
+|VMware Tanzu Kubernetes Grid
```

Comment:
I don't this is relevant to ARM orchestrators. The QA mentioned that this is for amd64 and not ARM

Reply:
@MorShabiPAN Yes, I just met with QA and they confirmed and clarified.

Reply:
Updated the table for clarity.